source/module_esolver/esolver_lj.cpp

```diff
@@ -54,12 +162,13 @@ namespace ModuleESolver
                 for (int ad = 0; ad < grid_neigh.getAdjacentNum(); ++ad)
                 {
                     tau2 = grid_neigh.getAdjacentTau(ad);
+                    int it2 = grid_neigh.getType(ad);
                     dtau = (tau1 - tau2) * ucell.lat0;
```

Comment:
is this PBC-safe? say one direct coord as 0 the other 1.1, will the distance be 0.1 rather than 1.1?

Reply:
It is the algorithm implemented in module_neighbor, which is used in lcao basis sets calculations. So I think it is safe.